packages/react/scroll-area/src/ScrollArea.tsx

```diff
@@ -196,17 +196,12 @@ ScrollAreaViewport.displayName = VIEWPORT_NAME;
 
 const SCROLLBAR_NAME = 'ScrollAreaScrollbar';
 
-type ScrollAreaScrollbarOwnProps =
-  | Polymorphic.OwnProps<typeof ScrollAreaScrollbarAuto>
-  | Polymorphic.OwnProps<typeof ScrollAreaScrollbarHover>
-  | Polymorphic.OwnProps<typeof ScrollAreaScrollbarScroll>
-  | Polymorphic.OwnProps<typeof ScrollAreaScrollbarVisible>;
-
+type ScrollAreaScrollbarOwnProps = Polymorphic.Merge<
+  Polymorphic.OwnProps<typeof ScrollAreaScrollbarVisible>,
```

Comment:
They all eventually render a `ScrollAreaScrollbarVisible`. Not ideal that we have to type this assumption here but not sure what else we can do.

Reply:
ok so we end up with simplified types whilst slightly inaccurate in the sense that if you use the visible one, TS will let you add a forceMount prop without complaints even though it doesn't apply?

Reply:
If you use `type="always"` it will say it accepts a `forceMount` prop which is kinda irrelevant there because `type="always"` is always mounted by nature of the type haha. 

So yeah, it will accept the `forceMount` prop but do nothing with it.